src/prettytable/prettytable.py

```diff
@@ -56,6 +56,7 @@
 PLAIN_COLUMNS = 12
 MARKDOWN = 13
 ORGMODE = 14
+DRAWING = 15
```

Comment:
```suggestion
DRAWING = 16
```

https://github.com/jazzband/prettytable/pull/104 will be 15, so let's go for 16.

Please could you also add a test case so the new code is covered?

Reply:
Sure, i've done the changes and run the test - should work now